### ISSUE TITLE
comparing GMAPS naming of cities to know cities in colibris zones

### ISSUE THREAD

**Issue by mghlaiel** [creometry-incubator/colibris#43]
Google maps uses crowdsourcing to register city names. This results in the same city name written in multiple different ways and does not compare with known city names in the colibris zoning table

**Comment by mghlaiel** [comment]
I believe The eligibility service does NOT require an exact address, which means we can do it without using any GMaps APIs.
Instead we can simply ask the the customer to choose a Gouvernerat and City from a drop-down list and match it against the Colibris schedule DB (the table defined by Selim which should be stored in DB and updated via the admin's dashboard).
When an already registered customer is adding a new address (i.ePick-up location) to their account, we can still rely on the value of the field city selected from the drop-down list to check its eligibility. Then, the use of GMaps Geocoding to get the Long/Lat serves only to replace the Street Address, making this field (Street Address) optional in the Pick-up location object.
Basically:
The eligibility check becomes much simpler by disregarding the use of GMaps and taking a city name to compare against a pre-defined list of city/zones.
A list of Cities by Gouvernerat is needed to be displayed in drop-down lists for Guest customers (Non-authenticated) and for registered customers who are adding a new or modifying an existing address.
A light-weight DB (e.g. SqLite) can store the Colibris schedule, keeping the eligibility check up-to-date with modifications made by colibris.
Please don't hesitate to punch a whole in this reasoning, as I might be missing something.
